util/chplenv/chpl_llvm.py

```diff
@@ -15,7 +15,7 @@ def llvm_versions():
     # Which major release - only need one number for that with current
     # llvm (since LLVM 4.0).
     # These will be tried in order.
-    return ('15','14','13','12','11',)
+    return ('16','15','14','13','12','11',)
```

Comment:
Let's leave this change out for now, and add it back in just before upgrading the bundled LLVM.

Reply:
I restored this, putting '16' at the end of the list so it gets tried last